components/service/glean/src/main/java/mozilla/components/service/glean/Dispatchers.kt

```diff
@@ -64,6 +94,20 @@ internal object Dispatchers {
         fun setTestingMode(enabled: Boolean) {
             testingMode = enabled
         }
+
+        /**
+         * Handle initialization complete and process any queued tasks
+         */
+        fun onInitComplete() {
+            // Setting this to false first should cause any new tasks to just be executed (see
+            // launch() above) making it safer to process the queue.
+            queueTasks = false
```

Comment:
If some task gets dispatched after this point but before the loop below, when would they be executed?

Reply:
Yes, if launch() was called after queueTask=false and before the loop, then it would just be executed as normal.